x-pack/plugins/observability_solution/infra/public/pages/metrics/inventory_view/components/bottom_drawer.tsx

```diff
@@ -117,19 +117,19 @@ export const BottomDrawer = ({ interval, formatter, view, nodeType }: Props) =>
   );
 };
 
-const BottomActionContainer = euiStyled.div`
+const BottomActionContainer = styled.div`
   position: sticky;
   bottom: 0;
   left: 0;
-  background: ${(props) => props.theme.eui.euiColorGhost};
-  width: calc(100% + ${(props) => props.theme.eui.euiSizeL} * 2);
-  margin-left: -${(props) => props.theme.eui.euiSizeL};
+  background: ${(props) => props.theme.euiTheme.colors.ghost};
```

Comment:
Question: Is it expected that this container always has a white background, also in dark mode?
If it should update the background based on `colorMode` (having a dark background in dark mode) then this should rather use `backgroundBasePlain`.

Reply:
I will change it as ghost is deprecated already

Reply:
>it already looks dark in dark mode though
I had a look and it is white on dark mode but it's using an `EuiPanel` as content which actually updates colors.

If this background color for the sticky container is not needed, we could also remove it.

![Screenshot 2024-12-05 at 12 59 29](https://github.com/user-attachments/assets/bfc12385-35c9-4da0-afc9-94d331f283c2)